packages/remix-dev/cli/migrate/migrations/convert-to-javascript/index.ts

```diff
@@ -0,0 +1,48 @@
+import { join } from "path";
+import glob from "fast-glob";
+
+import { readConfig } from "../../../../config";
+import * as jscodeshift from "../../jscodeshift";
+import type { MigrationFunction } from "../../types";
+import { cleanupPackageJson } from "./cleanupPackageJson";
+import { convertTSConfigs } from "./convertTSConfigs";
+import { convertTSFilesToJS } from "./convertTSFilesToJS";
+
+const TRANSFORM_PATH = join(__dirname, "transform");
+
+export const convertToJavascript: MigrationFunction = async ({
+  flags,
+  projectDir,
+}) => {
+  // 1. Rename all tsconfig.json files to jsconfig.json
+  convertTSConfigs(projectDir);
+
+  // 2. Remove @types/* & TypeScript dependencies + typecheck script from package.json
+  await cleanupPackageJson(projectDir);
+
+  // 3. Run codemod
+  let config = await readConfig(projectDir);
+  let files = glob.sync("**/*.+(ts|tsx)", {
+    absolute: true,
+    cwd: config.appDirectory,
+    ignore: ["./app/**/*"],
```

Comment:
might want to also use the appDirectory here
```suggestion
    ignore: [`./${path.dirname(config.appDirectory)}/**/*`],
```